resources/tag_models/page.json

```diff
@@ -0,0 +1,57 @@
+{
+  "tag": {
+	"api_key": "", // String - API key - Make sure the request is from a valid client (idea nicked from Keen.io) useful if a page gets copied onto a Russian website and creates noise
+	"version": "", // Version of the tracking client e.g. "JS v1.2"
+	"type": "page", // Type of request (page or event)
+	"id": "", // ID of this request, must be unique
+	"offset": 0 // Delay of this tag between event happening and being sent to server - milliseconds
```

Comment:
offset is used by the web app for tracking things which happen offline.  We don't want to trust the client's absolute time.  Instead we work out the difference in time between the event happening and it getting sent to the server.  That way we can subtract it from the time on the server and work out when the event took place.